plugins/modules/win_firewall.py

```diff
@@ -46,6 +58,12 @@
     profiles:
     - Domain
   tags: disable_firewall
+- name: Enable firewall for Domain profile and block outbound connections
+  win_firewall:
```

Comment:
Examples should use the fully qualified module name

```suggestion
  community.windows.win_firewall:
```